package.json

```diff
@@ -22,6 +22,7 @@
     "test": "pnpm jest",
     "jest": "jest -c jest.config.js --",
     "jest:e2e": "jest -c jest.config.e2e.js --",
+    "jest:e2e-all": "cross-env e2e_all=1 jest -c jest.config.e2e.js -- && cross-env parser_e2e=1 jest -c jest.config.e2e.js --forceExit -- ",
```

Comment:
I don't understand this change.  First, can we move the environment variables into the Jest setup vs. here?  Also, I think it's fine to add `--forceExit` to the regular `jest:e2e` call.

Reply:
ahh that's even better then